Source/WebCore/platform/cocoa/VideoPresentationModelVideoElement.mm

```diff
@@ -101,6 +101,8 @@
     if (m_videoElement) {
         for (auto& eventName : observedEventNames())
             m_videoElement->addEventListener(eventName, m_videoListener, false);
+        for (auto& eventName : documentObservedEventNames())
+            m_videoElement->document().addEventListener(eventName, m_videoListener, false);
```

Comment:
I think this is missing a matching call in `void VideoPresentationModelVideoElement::cleanVideoListeners()`, to `removeEventListener()` for each of the `documentObservedEventNames()`.

Reply:
Yes it is, thanks

Reply:
Just added this